stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -32,6 +34,8 @@
 
   private final Context mContext;
 
+  private Map<String, AsyncPrettyPrinter> mRequestIdMap = new HashMap<>();
```

Comment:
this might be accessed by multiple threads? if so, let's make sure we `Collections.synchronizedMap()` it